src/webhook/mutation/pod_mutator.go

```diff
@@ -794,11 +803,18 @@ func updateContainerDataIngest(c *corev1.Container, pod *corev1.Pod, deploymentM
 func getResponseForPod(pod *corev1.Pod, req *admission.Request) admission.Response {
 	marshaledPod, err := json.MarshalIndent(pod, "", "  ")
 	if err != nil {
-		return admission.Errored(http.StatusInternalServerError, err)
+		return silentErrorResponse(pod.Name, err)
 	}
 	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledPod)
 }
 
 func fieldEnvVar(key string) *corev1.EnvVarSource {
 	return &corev1.EnvVarSource{FieldRef: &corev1.ObjectFieldSelector{FieldPath: key}}
 }
+
+func silentErrorResponse(podName string, err error) admission.Response {
+	rsp := admission.Patched("")
+	message := fmt.Sprintf("Failed to inject into pod: %s because %s", podName, err.Error())
+	rsp.Result.Message = message
```

Comment:
can't you set rsp.Result.Message with the error message directly instead of passing it to the intermediate variable?